src/lib/awsElasticSearch.js

```diff
@@ -0,0 +1,222 @@
+/* eslint-disable camelcase */
+import { Client, Connection } from '@opensearch-project/opensearch';
+import aws4 from 'aws4';
+import { auditLogger, logger } from '../logger';
+
+const generateEsConfig = () => {
+  // Pull from VCAP env variables (cloud.gov)
+  if (process.env.VCAP_SERVICES) {
+    const {
+      'aws-elasticsearch': [{
+        credentials: {
+          uri,
+          access_key,
+          secret_key,
+        },
+      }],
+    } = JSON.parse(process.env.VCAP_SERVICES);
+
+    return {
+      uri,
+      access_key,
+      secret_key,
+    };
+  }
+
+  // Return docker image credentials.
+  return {
+    uri: 'http://localhost:9200',
+    access_key: 'admin',
+    secret_key: 'admin',
```

Comment:
Can we move these into env variables so the app is more configurable?

Reply:
Moved to env vars.